README.md

```diff
@@ -1,58 +1,53 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# Wemdigo
+- A ruby on rails app for keep track of your expenses. Users can create categories and transactions to which.
+- Notice: This is a mobile app so open it with your phone or make sure you are using a 400px width screen or resize the view using browser's inspection.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Ruby
+- RoR
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Live Demo
+- [Click Me!](https://desolate-stream-21599.herokuapp.com/)
```

Comment:
- [ ] One part of the project requirements expects that the project should be deployed and accessible online. It seems that this project was deployed but really not available. This is what we get when we access the live link:

  ![image](https://user-images.githubusercontent.com/45002654/153518668-02c44e66-9fc9-4876-b0b2-7cb900b1eb28.png)
  
  Please try and work on the deployment and provide a working link.

Reply:
I know right, It's really strange this is not the first time I deploy and we deployed last week too. I re-deployed again and the new link seems way better and faster and no bugs such as pictures not being loaded etc..

https://warm-fortress-20781.herokuapp.com/

I literally did the same thing again and now it's much better.

Reply:
This change has been implemented well